doc/designs/slh-dsa.md

```diff
@@ -90,9 +90,8 @@ Pre Hash SLH-DSA Signature Generation encode the message as
 The scenario that is stated that this is useful for is when this encoded message
 is supplied from an external source.
 
-Currently I do not support the Pre Hash variant as this does not sit well with the
-OpenSSL API's. The user could do the encoding themselves and then set the settable
-to not encode the passed in message.
+Currently we do not support the Pre Hash variant as this does not sit well with the
+OpenSSL API's.
```

Comment:
NIST uses _pre-hash_ as the variant name.  I suggest we do too.

Reply:
Not going to change this,...